common/membership/resolver.go

```diff
@@ -193,6 +198,7 @@ func (rpo *MultiringResolver) Unsubscribe(service string, name string) error {
 func (rpo *MultiringResolver) Members(service string) ([]HostInfo, error) {
 	ring, err := rpo.getRing(service)
 	if err != nil {
+		rpo.metrics.Scope(metrics.ResolverHostNotFoundScope).IncCounter(1)
```

Comment:
Isn't it better to move to LookupByAddress() which calls `Members`?
Then it would be clear - every time `LookupByAddress` returns empty `HostInfo` we emit the metric.

Reply:
yes, you're right, I was being a bit hurried and this doesn't make sense